src/Monitor/Monitor/help/Get-AzLog.md

```diff
@@ -213,7 +213,6 @@ Accept wildcard characters: False
 
 ### -CorrelationId
 Specifies the correlation ID.
-This parameter is required.
```

Comment:
this parameter seems required
[Mandatory = true](https://github.com/Azure/azure-powershell/blob/master/src/Monitor/Monitor/ActivityLogs/GetAzureRmLogCommand.cs#L68)

Reply:
But it is not. Look at the examples.

Reply:
Can you point which example indicates "-CorrelationId" is not required?

Reply:
Get-AzLog -MaxRecord 100
Get-AzLog -ResourceId $ResourceId -MaxRecord 5
Get-AzLog -DetailedOutput -ResourceId $ResourceId -Caller '*@dnhf.onmicrosoft.com'

All of these works fine.

Reply:
cmdlets can have multiple parameter sets,
CorrelationId is required in parameter set "CorrelationIdParameterSetName", other parameter sets could work fine without it.

Reply:
An alias would be nice :-)

Reply:
I'll close this PR, can you raise another issue regarding the cmdlet name so we can track it? Simply specify the command name in CLI / Portal / desired alias would be enough, thanks.

Reply:
Ok. But when I look at the documentation as it is now:
https://docs.microsoft.com/en-us/powershell/module/az.monitor/get-azlog?view=azps-3.4.0
Then try to look for the word required. It is only mentioned once on the page. 
That does not make any sense to me?

Reply:
Ok, this is the last time I will ever make any contribution to the Azure Powershell module.
I waste my weekend trying to make it better and it is just closed!! No wonder the documentation is so shit.

Reply:
Hi @theheatDK , 
this PR is to make change the description of parameter "-CorrelationId" which is incorrect as we discussed.

We appreciate your passion and effort to help make azure-powershell better, however to have an alias for this cmdlet, we need a  dedicated issue to track it.

About this help doc, it seems it was not generated properly, thanks for point it out, we'll regenerate doc for monitor in the future releases.

Reply:
I wrote this:

> The -CorrelationId parameter is not a required parameter.
> 
> It is not mentioned anywhere that the module retrieves Activity Log events. It is nice with a bit of context. In the Portal it is called Activity Log and the CLI uses "az monitor activity-log".

I made 3 changes to the text. One regarding the parameter and two to make it clear that this cmdlet is used for Activity logs. I would have assumed that you looked at my changes before dismissing them.

Reply:
Reopened PR,
Can you add the "This parameter is required." back, and something more meaningful in the ChangeLog.md? Something like "fix description of cmdlet, thanks.

Reply:
Hi @VeryEarly,
I have added the text back and changed the text in the ChangeLog.md.
Regards Peter